src/Microsoft.SqlTools.ServiceLayer/DacFx/DacFxService.cs

```diff
@@ -26,8 +26,8 @@ namespace Microsoft.SqlTools.ServiceLayer.DacFx
     /// </summary>
     class DacFxService
     {
-        private static ConnectionService connectionService = null;
-        private SqlTaskManager sqlTaskManagerInstance = null;
+        private static ConnectionService? connectionService = null;
```

Comment:
Is it actually expected that these will be null? Shouldn't that be something that's set up when this is created?

Reply:
removed them since they weren't really needed. Not sure why I added them 4 years ago haha